hikari/impl/shard.py

```diff
@@ -861,7 +856,7 @@ async def _run_once(self) -> bool:
 
         try:
             # Dispatch CONNECTED synthetic event.
-            self._event_manager.dispatch(self._event_factory.deserialize_connected_event(self)),
```

Comment:
Isn't this inconsistent compared with how artificial events are dispatched in bot.py
(also the event manager doesn't have managers for any artificial events anymore)

Reply:
`bot.py` is able to dispatch artificial events due to it being the "owner". As shard is not the "owner", it is not able to do this and has to rely on the implementation of `bot.py`

Reply:
So the shards are meant to just blindly dispatch an fake raw event which may or may not be dispatched based on implementation detail separate from them?

Reply:
This feels less like a case of "ownership" and more like a case of ensuring the events are actually dispatched which you cannot do with the consume raw event method